packages/nitro-protocol/test/contracts/ninja-nitro/SingleChannelAdjudicator/transfer.test.ts

```diff
@@ -60,26 +62,31 @@ const reason1 = 'Indices must be sorted';
+    ${' 1. funded          -> 1 EOA'}  | ${{c: 1}}  | ${{A: 1}}             | ${[0]}       | ${{A: 0}}             | ${{c: 0}} | ${{A: 1}}             | ${undefined}
+    ${' 2. overfunded      -> 1 EOA'}  | ${{c: 2}}  | ${{A: 1}}             | ${[0]}       | ${{A: 0}}             | ${{c: 1}} | ${{A: 1}}             | ${undefined}
+    ${' 3. underfunded     -> 1 EOA'}  | ${{c: 1}}  | ${{A: 2}}             | ${[0]}       | ${{A: 1}}             | ${{c: 0}} | ${{A: 1}}             | ${undefined}
+    ${' 4. funded      -> 1 channel'}  | ${{c: 1}}  | ${{C: 1}}             | ${[0]}       | ${{C: 0}}             | ${{c: 0}} | ${{C: 1}}             | ${undefined}
+    ${' 5. overfunded  -> 1 channel'}  | ${{c: 2}}  | ${{C: 1}}             | ${[0]}       | ${{C: 0}}             | ${{c: 1}} | ${{C: 1}}             | ${undefined}
+    ${' 6. underfunded -> 1 channel'}  | ${{c: 1}}  | ${{C: 2}}             | ${[0]}       | ${{C: 1}}             | ${{c: 0}} | ${{C: 1}}             | ${undefined}
+    ${' 7. -> 2 EOA         1 index'}  | ${{c: 2}}  | ${{A: 1, B: 1}}       | ${[0]}       | ${{A: 0, B: 1}}       | ${{c: 1}} | ${{A: 1}}             | ${undefined}
+    ${' 8. -> 2 EOA         1 index'}  | ${{c: 1}}  | ${{A: 1, B: 1}}       | ${[0]}       | ${{A: 0, B: 1}}       | ${{c: 0}} | ${{A: 1}}             | ${undefined}
+    ${' 9. -> 2 EOA         partial'}  | ${{c: 3}}  | ${{A: 2, B: 2}}       | ${[1]}       | ${{A: 2, B: 1}}       | ${{c: 2}} | ${{B: 1}}             | ${undefined}
+    ${'10. -> 2 chan             no'}  | ${{c: 1}}  | ${{C: 1, X: 1}}       | ${[1]}       | ${{C: 1, X: 1}}       | ${{c: 1}} | ${{}}                 | ${undefined}
+    ${'11. -> 2 chan           full'}  | ${{c: 1}}  | ${{C: 1, X: 1}}       | ${[0]}       | ${{C: 0, X: 1}}       | ${{c: 0}} | ${{C: 1}}             | ${undefined}
+    ${'12. -> 2 chan        partial'}  | ${{c: 3}}  | ${{C: 2, X: 2}}       | ${[1]}       | ${{C: 2, X: 1}}       | ${{c: 2}} | ${{X: 1}}             | ${undefined}
+    ${'13. -> 2 indices'}              | ${{c: 3}}  | ${{C: 2, X: 2}}       | ${[0, 1]}    | ${{C: 0, X: 1}}       | ${{c: 0}} | ${{C: 2, X: 1}}       | ${undefined}
+    ${'14. -> 3 indices'}              | ${{c: 5}}  | ${{A: 1, C: 2, X: 2}} | ${[0, 1, 2]} | ${{A: 0, C: 0, X: 0}} | ${{c: 0}} | ${{A: 1, C: 2, X: 2}} | ${undefined}
+    ${'15. -> reverse order (see 13)'} | ${{c: 3}}  | ${{C: 2, X: 2}}       | ${[1, 0]}    | ${{C: 2, X: 1}}       | ${{c: 2}} | ${{X: 1}}             | ${reason1}
   `(
     `$name: heldBefore: $heldBefore, setOutcome: $setOutcome, newOutcome: $newOutcome, heldAfter: $heldAfter, payouts: $payouts`,
     async ({name, heldBefore, setOutcome, indices, newOutcome, heldAfter, payouts, reason}) => {
+      // Extraneous keys would make sense in a monolithic Nitro implementation (e.g. in an AssetHolder governing multiple channels)
```

Comment:
Note to next reviewer: scope creep here - this one-liner is unrelated to the bytes32 -> address issue.

Reply:
I think it is not really unrelated. Part of the reason this test did not pick up the bug fixed in this PR is that it was still expressing the expected outcome of the `transfer` transaction in the "old" way. (The "old" way being, a contract having holdings for several channels). This line explains why the test will now fail if it were to do that.

Reply:
Is there a change in variable name or comment that would clarify that in the code? I found I had to think pretty hard to grasp the relevance.